packages/node_modules/@webex/plugin-meetings/src/meeting-info/utilv2.js

```diff
@@ -116,8 +118,32 @@ MeetingInfoUtil.getSipUriFromHydraPersonId = (destination, webex) => webex.peopl
 
 
 MeetingInfoUtil.getDestinationType = async (from) => {
-  const {destination, type, webex} = from;
+  const {type, webex} = from;
+  let {destination} = from;
+
+  if (type === _PERSONAL_ROOM_) { // this case checks if your type is personal room
```

Comment:
This changes look like they're in the wrong place. Shouldn't they be part of `MeetingInfoUtil.getRequestBody` instead?
That's where the org ID and user ID values need to be figured out

Reply:
@arun3528  not sure of this one .

Reply:
We'll leave in on `getDestinationType` for now